src/Apps/Collections/routes.tsx

```diff
@@ -0,0 +1,25 @@
+import { RouteConfig } from "found"
+import React from "react"
+import { graphql } from "react-relay"
+import { CollectionsAppFragmentContainer as CollectionsApp } from "./CollectionsApp"
+
+export const routes: RouteConfig[] = [
+  {
+    path: "/collections",
+    Component: CollectionsApp,
+    query: graphql`
+      query routes_MarketingCollectionsAppQuery {
+        collections: marketingCollections {
+          ...CollectionsApp_collections
+        }
+      }
+    `,
+    render: ({ props, Component }) => {
+      if (!props) {
```

Comment:
I could be wrong, but I think `props` always evaluates to `{}` or an object that has properties at runtime. Is this check really necessary?

Reply:
Yeah it should be there, when `props` is null, it means relay hasn't gotten any data back yet so we shouldn't render the relay container. We could actually have a `<Loading />` instead of `null` here but since we're using SSR, it's not needed.

See https://github.com/relay-tools/found-relay#render